### PR TITLE
Update migrate-from-e3.md

### PR DESCRIPTION
**Pull request by msbemba**
Added section for mailbox size as they are downgrading from 100Gb to 50Gb

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2911

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/business/migrate-from-e3.md [hunk 1]
```diff
@@ -109,6 +109,11 @@ You can also complete these steps on the Devices page:
   - See [Set up Windows devices for Microsoft 365 Business Premium users](set-up-windows-devices.md) for steps for Windows devices. 
     
   - See [Set up mobile devices for Microsoft 365 Business Premium users](set-up-mobile-devices.md) for steps for Android phones and iPhones. 
+  
+### Mailbox Size
+
+Microsoft 365 Business premium has 50Gb storage limit as it uses Exchange Online Plan 1. While Migrating to Microsoft 365 Business Premium, If any of your users are exceeding 50GB mailbox storage  it is recommended that you assign this user Exchange Plan 2 and remove the Exchange Online Plan 1 as It's not feasible to assign both.
```
Comment (JohanFreelancer9) [inline microsoft-365/business/migrate-from-e3.md]:
```suggestion
Microsoft 365 Business Premium has a 50 GB storage limit as it uses Exchange Online Plan 1. While migrating to Microsoft 365 Business Premium, if any of your users exceed 50 GB of mailbox storage, it is recommended that you assign this user an Exchange Online Plan 2 and remove the Exchange Online Plan 1 as it's not feasible to assign both.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks.

**Pull request merged**